server/src/modules/core/usecases/anonymiseUsers/anonymiseUsers.dep.ts

```diff
@@ -0,0 +1,36 @@
+import { sql } from "kysely";
+
+import { getKbdClient } from "@/db/db";
+
+export const countUsersToAnonymiseQuery = async () =>
```

Comment:
```suggestion
export const countUsersToAnonymizeQuery = async () =>
```

Ou alors l'écrire avec un `S` mais choisir entre les deux ;)